data-serving/scripts/prune-uploads/hooks/aggregate.py

```
@@ -26,7 +26,7 @@ def run(sources: list[dict[str, Any]], env: str, dry_run: bool = False):
         )
     except Exception as e:
         logging.exception("Error occurred when fetching job definitions")
-        return
+        return "*Failed* to start aggregation job"
```

`raise` here instead of `return`?